runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -706,7 +827,11 @@ void pauseSkill(Skill skill, boolean pause)
 	{
 		if (pause ? xpPauseState.pauseSkill(skill) : xpPauseState.unpauseSkill(skill))
 		{
-			xpPanel.updateSkillExperience(false, xpPauseState.isPaused(skill), skill, xpState.getSkillSnapshot(skill));
+			final boolean updated = false;
+			final boolean paused = xpPauseState.isPaused(skill);
+			final XpSnapshotSingle xpSnapshotSingle = xpState.getSkillSnapshot(skill);
+
+			xpPanel.updateSkillExperience(updated, paused, skill, xpSnapshotSingle);
```

Comment:
Same here. Your IDE should show you the argument names for this method, I don't see any value in assigning variables here.